### ISSUE TITLE
Use of unresolved identifier 'CBC' - build fails

### ISSUE THREAD

**Issue by wingsofovnia** [BANKEX/web3swift#135]
Commit https://github.com/BANKEX/web3swift/commit/8bbde2ea49d02cf98ae16c5717089f8b6bac8851 leads to build failure:
```
BIP32Keystore.swift:165:71: use of unresolved identifier 'CBC'
EthereumKeystoreV3.swift:90:71: use of unresolved identifier 'CBC'
....
```
Such dependency works tho (points to a commit before the above one):
```
pod 'web3swift', :git => 'https://github.com/BANKEX/web3swift.git', 
                     :commit => '8d7a985f93b3a5259366ae00606ff622cbc7fe5f' # Next commit breaks the build
```

**Comment by shamatar** [comment]
0.8 tag is buildable, please specify a tag version in your podfile.

________________________________

**Comment by runryan** [comment]
I met the same error. I've solved this by following @shamatar 's instruction.
```
pod 'web3swift', :git => 'https://github.com/BANKEX/web3swift.git', :tag => '0.8.0'
```

**Comment by MrShafiee** [comment]
Hi @runryan, why did in encryptDataToStorage function on BIP32keystore.swift you set cipher : "aes-128-cbc" static! I want use it  as "aes-128-ctr" and I should edit pod library! Can I sent it via parameters?

<img width="824" alt="screen shot 2018-06-10 at 3 19 51 pm" src="https://user-images.githubusercontent.com/19608549/41200871-c46723de-6cc1-11e8-9cc2-a675919f2cda.png">

**Comment by shamatar** [comment]
Regarding the original issue - it should be fixed in 0.8.1 with CryptoSwift dependency bumped and written in requirements.

@MrShafiee 

This behavior corresponds to one in original go-ethereum implementation for plain keystores (KeystoreV3 in web3swift). There is no standard for HD (BIP32) key trees storage, so in principle it can be changed. I'll add this option eventually, please write the corresponding feature request in separate issue.

Sincerely, Alex
